### ISSUE TITLE
Accessibility checklist review

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1326]
@aschmoe good to review this to make sure we're not missing anything, especially #1. 

https://www.ada.gov/pcatoolkit/chap5chklist.htm

**Comment by lukefretwell** [comment]
@kevindherman see #16 re: documents:

> 16.    When documents are added to your website in PDF format, are text-based versions of the documents (e.g., HTML, RTF, or word processing format) added at the same time as the PDF versions?

**Comment by kevindherman** [comment]
@lukefretwell I think this is a good question for Lindsay. I don't want to force something that conflicts with requirements or at least not know what her requirements are  before making decisions.
